tests/ui_tests/test_cluster_list_page.py

```diff
@@ -209,6 +210,52 @@ def check_components_host_info(host_info: ComponentsHostRowInfo, name: str, comp
     check_host_value('components', host_info.components, components)
 
 
+# !===== Funcs =====!
+
+
+def cluster_with_service(sdk_client: ADCMClient, path) -> Cluster:
+    """
+    Prepared cluster for test: create cluster, couple services and couple components.
+    """
+    bundle = sdk_client.upload_from_fs(path)
+    cluster = bundle.cluster_create(name=f"Test cluster {random_string()}")
+    return cluster
```

Comment:
No services are added to the cluster here **and** path is passed as an argument, so it's neither "bundle where cluster have services", so I suggest adding services here or rename function

Reply:
changed